native/java/org/jpype/JPypeContext.java

```diff
@@ -305,4 +307,23 @@ public Exception createException(long l0, long l1)
     return new PyExceptionProxy(l0, l1);
   }
 
+  public boolean order(Buffer b)
+  {
+    if (b instanceof java.nio.ByteBuffer)
+      return ((java.nio.ByteBuffer) b).order() == ByteOrder.LITTLE_ENDIAN;
+    if (b instanceof java.nio.ShortBuffer)
+      return ((java.nio.ShortBuffer) b).order() == ByteOrder.LITTLE_ENDIAN;
+    if (b instanceof java.nio.CharBuffer)
+      return ((java.nio.CharBuffer) b).order() == ByteOrder.LITTLE_ENDIAN;
+    if (b instanceof java.nio.IntBuffer)
+      return ((java.nio.IntBuffer) b).order() == ByteOrder.LITTLE_ENDIAN;
+    if (b instanceof java.nio.LongBuffer)
+      return ((java.nio.LongBuffer) b).order() == ByteOrder.LITTLE_ENDIAN;
+    if (b instanceof java.nio.FloatBuffer)
+      return ((java.nio.FloatBuffer) b).order() == ByteOrder.LITTLE_ENDIAN;
+    if (b instanceof java.nio.DoubleBuffer)
+      return ((java.nio.DoubleBuffer) b).order() == ByteOrder.LITTLE_ENDIAN;
```

Comment:
This code was just miserable.   For reasons I find unfathomable there is no order on the base class so every single class needs its own lookup.  Thus better to do this in Java because otherwise we have to lookup the methodID over and over.

Reply:
That's indeed tedious... Agreed to have it on the java side